strr-web/components/bcros/form-section/principal-residence/Form.vue

```diff
@@ -91,8 +91,7 @@
                 class="text-bcGovColor-activeBlue"
                 target="_blank"
                 :href="`
-                  https://www2.gov.bc.ca/gov/content/housing-tenancy/\
-                    short-term-rentals/principal-residence-requirement
+                  https://www2.gov.bc.ca/gov/content/housing-tenancy/short-term-rentals/principal-residence-requirement
```

Comment:
If a static link you can remote the `:` and back ticks.

Reply:
Fixed

Reply:
Almost... back ticks ` ` needs to be removed.